css/style.css

```diff
@@ -0,0 +1,17 @@
+html,
+body {
+  height: 100vh;
+}
+
+#welcome-section {
+  padding-top: 0 !important;
+  margin-top: 0 !important;
```

Comment:
- [ ] You should try as much as possible not to overuse the `!important` rule in your CSS style definitions.
Using `!important` is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets